src/Platform/Bridge/Bedrock/Anthropic/ClaudeHandler.php

```diff
@@ -73,7 +73,7 @@ public function convert(InvokeModelResponse $bedrockResponse): LlmResponse
                 $toolCalls[] = new ToolCall($content['id'], $content['name'], $content['input']);
             }
         }
-        if (!empty($toolCalls)) {
+        if (!(!$toolCalls)) {
```

Comment:
```suggestion
        if (!$toolCalls) {
```
?

Reply:
Of course! I missed this one. These AI tools have still some way to go... :)